hpx/util/batch_environments/pbs_environment.hpp

```diff
@@ -53,7 +31,7 @@ namespace hpx { namespace util { namespace batch_environments {
 
         std::size_t num_threads() const
         {
-            return num_threads_;
+            return std::size_t(-1);
```

Comment:
I removed it because we were handling the list of nodes provided as a nodefile differently from handling a list of nodes given on the command line. The further interpreted the number of times a node was given as the number of threads to use (which - I think - is questionable), the latter was not doing this at all. I'd rather either let the user explicitly specify the number of threads to run on or to look at the `PBS_NUM_VNODES` environment variable (or similar) to extract that number from PBS.